decode/lldp/ttl.js

```diff
@@ -0,0 +1,9 @@
+function ttl() {
+	this.ttl = undefined;	
+}
+
+chassisId.prototype.decode = function(tlv, raw_packet, offset) {
+	this.ttl = raw_packet.readUInt16BE(offset+1, true);
```

Comment:
change to tlv.ttlSeconds and there is no subType offset so just offset and not "offset+1"